app/views/search.scala.html

```diff
@@ -50,11 +51,17 @@
 	if(raw.contains("multiVolumeWork")) {select(("Band","Bände"))} else if (raw.contains("series")) {select(("Serienband","Serienbände"))} else if (raw.contains("containedIn")) {select(("Beitrag","Beiträge"))} else {""}
 }
 
-@nwbibLabel() = @{
-	if(!nwbibsubject.isEmpty && !nwbibsubject.contains(",") && Seq(nwbibspatial, person, name, subject, id, publisher, issued, medium, owner, t, set, location, word, corporation, raw).forall(_.isEmpty)){
-		s" in Sachsystematik zu '${Lobid.facetLabel(Seq(nwbibsubject),"","")}'"
```

Comment:
I know this is a very general question, but:
those long lines are not very convenient to read. What speaks against linebreaks after at least 100 chars?